### PR TITLE
Make better English example

### PR DESCRIPTION
**Pull request by askdkc**
- Fix sample to English friendly
- Add English version Screenshots

### CODE REVIEW

File: how-to/postgrest.md [hunk 1]
```diff
@@ -175,79 +165,72 @@ END;
 $$ LANGUAGE plpgsql;
 ```
 
+**NOTE: You need to restart `PostgREST`` when create a new functions**
+
+Before proceeding to the next section, please restart your `PostgREST`` by pressing `Ctrl + C`` to stop the currently running `PostgREST` instance, and then run it again using the following command:
```
Comment (kou) [inline how-to/postgrest.md:170]:
```suggestion
Before proceeding to the next section, please restart your PostgREST by pressing `Ctrl + C` to stop the currently running PostgREST instance, and then run it again using the following command:
```

File: how-to/postgrest.md [hunk 2]
```diff
@@ -552,10 +534,10 @@ Open `index.html` with your browser.
 
 Type something and it will show the suggestions.
 
-![PGroonga Auto Complete2](../images/postgrest/auto-complete2.png)
+![PGroonga Auto Complete2](../images/postgrest/auto-complete2_en.png)
```
Comment (kou) [inline how-to/postgrest.md:537]: Could you use `-` as word separator?

Comment (askdkc) [reply how-to/postgrest.md:537]: OK

**Comment by kou** [comment]
Thanks!

**Pull request merged**